Homework_4/commentify/src/helpers/sort.js

```diff
@@ -0,0 +1,16 @@
+const sort = (arr, dir = "descending") => {
```

Comment:
This part is tricky, the name suggest it should sort,  but apparently it also extends  Post object with average.
In task description I didn't notice that we can't add average rate directly to the post :), or at least we can run once on the initial array of the post, without modifying initial array, like const postsWithAverages = DUMMY_LIST.map(p=>{...p, average: "calculate it"}).

Reply:
Ok, will fix. But I didn't notice where is second PR. I have 1 PR and there is just 1 homework

Reply:
there isn't 2 pr, but this PR has  Homework 2 and 3 deleted in it, you can keep them